core/crypto/pedersen_hash.go

```diff
@@ -1,38 +1,84 @@
 package crypto
 
 import (
-	"github.com/NethermindEth/juno/core/crypto/starkware"
+	"math/big"
+
 	"github.com/NethermindEth/juno/core/felt"
+	"github.com/consensys/gnark-crypto/ecc/stark-curve"
+	"github.com/consensys/gnark-crypto/ecc/stark-curve/fp"
+)
+
+var (
+	shiftPoint  *starkcurve.G1Jac
+	p0          *starkcurve.G1Jac
+	p1          *starkcurve.G1Jac
+	p2          *starkcurve.G1Jac
+	p3          *starkcurve.G1Jac
+	lowPartMask [4]uint64
 )
```

Comment:
`lowPartMask` doesn't seem to be used. I think it should be removed if we aren't using it. Or we could do this
```suggestion
)

const (
	lowPartMask = 0xffffffffffffff
	lowPartBits = 56
)
```

then update then on lines [78](https://github.com/NethermindEth/juno/pull/476/files#diff-9ac8648882322077a92f77befbdec5aeee0a34a247faaea30e6a79a281fd1a55R78) and [81](https://github.com/NethermindEth/juno/pull/476/files#diff-9ac8648882322077a92f77befbdec5aeee0a34a247faaea30e6a79a281fd1a55R81)